src/backend/github/deployment/deployment.test.ts

```diff
@@ -1,13 +1,13 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
I'm not sure I like it that the test files now live in the same folders as the production code. This can lead to unwanted dependencies and mixups. I know this is kind of a default in Node apps, but my gut tells me it's better to have the tests in a separate folder structure. 

@mboudreau what's your experience with tests in their own folder structure in Node?

Reply:
I grouped files like this because frontend guidelines stipulate that this is the structure that should be followed. I know these are all backend files but it seems kind of messy to have the backend tests pulling living in /tests while the frontend tests are grouped with the source files. That said, I don't think it's common practice to group test files for node apps with source files 🤷  

When I chatted to some folks in the Fusion team they said all new frontend apps need to be created within the atlassian-frontend monorepo and follow the 'guidelines' (which are really more of a mandate. I told them we can't do this as our backend and frontend are grouped together and, because we have an open source app with such a tiny UI, it's unlikely that they'll split them. That got us out of living in the monorepo but they said we should try and follow the guides as much as possible.

Would love some ideas here.

Reply:
As I mentioned to Tom already, adding the tests next to prod code is a common practice with node apps.  It makes it much easier to update code and tests together without having to go looking in a different directory.  When we're building the typescript, we can just exclude `.test.ts` files so they're not built as part of the final package.  Leave all other commonalities in a `test` folder separate from `src`